databases/week1/index.sql

```diff
@@ -2,11 +2,19 @@
 SELECT COUNT(id) FROM task LIMIT 100;
 /*Find out how many tasks in the task table do not have a valid due date*/
 SELECT COUNT(id) FROM task WHERE `due_date` IS NULL LIMIT 100;
-Find all the tasks that are marked as done
-Find all the tasks that are not marked as done
-Get all the tasks, sorted with the most recently created first
-Get the single most recently created task
-Get the title and due date of all tasks where the title or description contains database
-Get the title and status (as text) of all tasks
+/* Find all the tasks that are marked as done */
+SELECT * FROM task JOIN status ON `status.id` = 'task.status_id' WHERE status.name = 'Done'
+
+/*Find all the tasks that are not marked as done*/
+
+SELECT * FROM task JOIN status ON status.id = task.status_id WHERE status.name != 'Done'
+/*Get all the tasks, sorted with the most recently created first*/
```

Comment:
You can use  ' --Get all the tasks, sorted with the most recently created first ' for single line comments